src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -24,23 +23,27 @@
      * Signals that an operation would have violated the 'no duplicates' property of the list.
      */
     public static class DuplicatePersonException extends DuplicateDataException {
+
         protected DuplicatePersonException() {
             super("Operation would result in duplicate persons");
         }
     }
 
     /**
-     * Signals that an operation targeting a specified person in the list would fail because
-     * there is no such matching person in the list.
+     * Signals that an operation targeting a specified person in the list would fail because there
```

Comment:
Hmm is this change really necessary? Try to make the least changes to achieve your goal every time in order to make your PR clean & clear.

Reply:
I think it accidentally auto formatted the whole file when scrolling through. Same reasons apply to the next few.